core/src/token_info.rs

```diff
@@ -102,4 +154,33 @@ mod tests {
         assert_eq!(TokenBaseInfo::new("", 1).base_unit_in_atoms().get(), 10);
         assert_eq!(TokenBaseInfo::new("", 2).base_unit_in_atoms().get(), 100);
     }
+
+    #[test]
+    fn default_get_token_infos_forwards_calls_and_ignores_errors() {
+        // Not using mockall because we want to test the default impl.
+        struct TokenInfo {};
```

Comment:
nit: could use the hardcoded `TokenData` here, but not depending on a concrete implementation is fine too.